identity-apps-core/apps/authentication-portal/src/main/webapp/domain.jsp

```diff
@@ -37,6 +38,8 @@
 
             if (domainUnknown.equalsIgnoreCase(errorMessage)) {
                 errorMessage = AuthenticationEndpointUtil.i18n(resourceBundle, "domain.cannot.be.identified");
+            } else if (idpNotAssignedToSP.equalsIgnoreCase(errorMessage)) {
```

Comment:
There's no need to modify translations for this key. Just directly comparing the i18n key with errorMessage is enough. 

Ref: 
https://github.com/wso2/identity-apps/blob/master/identity-apps-core/apps/authentication-portal/src/main/webapp/org_name.jsp#L49

Reply:
Resolved with https://github.com/wso2/identity-apps/pull/5322/commits/7891a913a44c8c6f267ed417e5b3c4ce633dc7a3